src/Compilers/CSharp/Test/Semantic/Semantics/StaticNullChecking_Fields.cs

```diff
@@ -0,0 +1,608 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests.Semantics
+{
+    public partial class StaticNullChecking : CSharpTestBase
```

Comment:
Nit: do you have a test trait for nullable feature?
`    [CompilerTrait(CompilerFeature.Tuples)]`